frontend/catalyst/autograph.py

```diff
@@ -134,13 +110,90 @@ def autograph(fn):
     return new_fn
 
 
+def autograph_source(fn):
+    """Utility function to retrieve the source code of a function converted by AutoGraph.
```

Comment:
Feels like something that would eventually make sense to upstream now!

Reply:
They do actually have their function for this `to_graph`, but the problem is it doesn't account for our decorators, for which we have special handling in a few locations.